examples/htlc-pyteal-ts/scripts/withdraw/common.ts

```diff
@@ -0,0 +1,30 @@
+import { executeTransaction } from "@algorand-builder/algob";
+import { Account, AlgobDeployer, ExecParams } from "@algorand-builder/algob/src/types";
+import { sha256 } from 'js-sha256';
+
+export function getDeployerAccount (deployer: AlgobDeployer, name: string): Account {
+  const account = deployer.accountsByName.get(name);
+  if (account === undefined) {
+    throw new Error(`Account ${name} is not defined`);
+  }
+  return account;
+}
+
+export async function executeTx (deployer: AlgobDeployer, txnParams: ExecParams): Promise<void> {
+  try {
+    await executeTransaction(deployer, txnParams);
+  } catch (e) {
+    console.error('Transaction Failed', e.response ? e.response.error : e.error);
+  }
+};
+
+export function prepareParameters (deployer: AlgobDeployer): any {
```

Comment:
please add doc string here

Reply:
can you please suggest something?

Reply:
Doc string explaining why are we using this function

Reply:
For example - preparing parameters and what are those parameters